test/upgrade/suite.go

```diff
@@ -57,3 +57,13 @@ func Suite() pkgupgrade.Suite {
 		},
 	}
 }
+
+func union(
```

Comment:
Hopefully, we won't get more than two slices :)
I'm curious if this could be even more general. I searched this code via Google but it could be something like it (just changing types...):
```
func mergeSlices(args...[]int) []int{
	mergedSlice := make([]int, 0)
	for _, oneSlice := range  args {
		mergedSlice = append(mergedSlice, oneSlice...)
	}
	return mergedSlice
}
```

Reply:
Downside of this implementation is allocation of new memory segment for each loop as capacity for slide is calculated automatically as `len(x)*2` - that's true for small slices.

Maybe this impl. would be better:

```golang
func merge(slices... []pkgupgrade.BackgroundOperation) []pkgupgrade.BackgroundOperation {
	l := 0
	for _, slice := range slices {
		l += len(slice)
	}
	result := make([]pkgupgrade.BackgroundOperation, 0, l)
	for _, slice := range slices {
		result = append(result, slice...)
	}
	return result
}
```

Reply:
Yeah. That looks good.